02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,114 @@
+
+
+
 <!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Nunito' rel='stylesheet' type='text/css'>
+    <title>Enrique Urbina</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
 
+    <!--
+<div id="bar">
+        <br>
+        <br>
+    </div>
+    <div>-->
+
+
+    <h1 id="name">Enrique Urbina</h1>
+
+    </div>
+    <img width="100%" height="40px" src="http://wallpoper.com/images/00/37/24/62/minimalistic_00372462.jpg" alt="Sorry:(">
+    <div id="location">
+        <h3>Location</h3>
+        <p>New Braunfels, TX, 78130</p>
+    </div>
+
+    <div id="about">
+
+        <h1 id="about2">About</h1>
+
+        <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Qui beatae, quod quos sunt laboriosam? Magnam facilis, non delectus. Veritatis, enim, perferendis. Earum at quae excepturi libero laboriosam, officiis quas, exercitationem? Lorem ipsum
```

Comment:
You should have written about yourself. The `Lorem ipsum` is only used during the early stages of development. The production ready work should have actual meaningful text. :smiley: